storage/entities/account.js

```diff
@@ -187,16 +187,6 @@ const readOnlyFields = ['address'];
  * @property {string} [votes_in]'
```

Comment:
Just taking the opportunity to remove this :)

```suggestion
  * @property {string} [votes_in]
```